src/Sulu/Bundle/PreviewBundle/Preview/Renderer/PreviewRenderer.php

```diff
@@ -158,6 +158,8 @@ public function render($object, $id, $webspaceKey, $locale, $partial = false, $t
         }
 
         $attributes = $this->routeDefaultsProvider->getByEntity(get_class($object), $id, $locale, $object);
+        $attributes['preview'] = true;
```

Comment:
@alexander-schranz i think this is the bug in master which should fix the preview and partial flag in the webiste controller

Reply:
I'm not sure about object and partial if they are needed? 
https://github.com/sulu/sulu/pull/4146/files

Reply:
partial is needed because of the `WebsiteController` to render only the content (without navigation, ...)

the object will be set by the `RouteDefaultsProvider` - so not needed to set afterwards